src/ts/pages/DataCatalog/__tests__/DataCatalog.test.tsx

```diff
@@ -106,6 +106,10 @@ const setup = ({ params }: { params?: string } = {}) => {
 describe('DataCatalog', () => {
   beforeEach(() => nock.cleanAll())
 
+  afterEach(() => {
+    vi.clearAllMocks()
```

Comment:
Was getting intermittent test failures from this. It seemed at least like mocks were not getting torn down